tools/topology/topology2/common_definitions.conf

```diff
@@ -58,4 +58,7 @@ Define {
 	SPI_INPUT_CLASS				26 # SPI Input (DSP <-)
 
 	DEEPBUFFER_FW_DMA_MS			100 # 100 ms copier dma size
+
+	SSP_BLOB_VERSION_1_0			0x100
+	SSP_BLOB_VERSION_1_5			0x105
```

Comment:
0x150 ?

Reply:
ack, this is 105 - as strange as it might look